metomi/rose/config.py

```diff
@@ -217,8 +217,8 @@ def __eq__(self, other):
         try:
             for keys_1, node_1 in self.walk(no_ignore=True):
                 node_2 = other.get(keys_1, no_ignore=True)
-                if (
-                    type(node_1) != type(node_2)
+                if (  # noqa: E721
+                    type(node_1) != type(node_2)  #  noqa: E721
```

Comment:
I think that this is what flake8 wants you to do...

```suggestion
                    type(node_1) != type(node_2)  # noqa: E721
```